src/components/ListEmployees.vue

```diff
@@ -0,0 +1,153 @@
+<template>
+  <div>
+    <h1>List Employees</h1>
+    <ModalComponent @closeModal="isOpen = false" @showModal="showModal" :isOpen="isOpen">
+      <table>
+        <thead>
+          <tr>
+            <th>Full Name</th>
+            <th>Age</th>
+            <th>Address</th>
+            <th>Date of birth</th>
+            <th>Company</th>
+            <th>Avatar</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td>
+              <input type="text" placeholder="Enter Fullname" />
+            </td>
+            <td>
+              <input type="number" placeholder="Enter Age" />
+            </td>
+            <td>
+              <input type="text" placeholder="Enter Address" />
+            </td>
+            <td>
+              <input type="date" placeholder="Enter Date of birth" />
+            </td>
+            <td>
+              <input type="text" placeholder="Enter Company" />
+            </td>
+            <td>
+              <input type="url" placeholder="Enter link avatar" />
+            </td>
+          </tr>
+        </tbody>
+      </table>
+      <br>
+      <button>NOT</button>
+      <button>OK</button>
+    </ModalComponent>
+    <br>
+    <table id="employees">
+      <thead>
+        <tr>
+          <th>Full Name</th>
+          <th>Age</th>
+          <th>Address</th>
+          <th>Date of birth</th>
+          <th>Company</th>
+          <th>Avatar</th>
+          <th>Action</th>
+        </tr>
```

Comment:
tương tự header ở đây anh thử for ra xem ntnao

Reply:
đã fix